ExpressBindingGenerator/src/Generator/GeneratorOIP.cpp

```diff
@@ -3534,7 +3556,7 @@ void GeneratorOIP::generateEntityHeaderFileREFACTORED(Schema & schema, Entity &
 	
 	//writeLine(out, "typedef " + entity.getName() + " UnderlyingType;");
 	
-	auto initClassAsDefault = [&out, &entity, &schema]() {

```

Comment:
Where did this go?

Reply:
I removed the function definition and just left the body. (The function was called once, directly after its definition).